imgui.cpp

```diff
@@ -157,13 +156,11 @@ CODE
 
  HOW TO UPDATE TO A NEWER VERSION OF DEAR IMGUI
  ----------------------------------------------
- - Overwrite all the sources files except for imconfig.h (if you have made modification to your copy of imconfig.h)
- - Or maintain your own branch where you have imconfig.h modified as a top-most commit which you can regularly rebase over master.
+ - Overwrite all the sources files except for imconfig.h (if you have modified your copy of imconfig.h)
+ - Or maintain your own branch where you have imconfig.h modified as a top-most commit which you can regularly rebase overmaster.
```

Comment:
Should be "over master" or "over master branch" for clarity, if you will.

Reply:
Oh, right, yea. My bad :sweat_smile:
This was modified in the merge, so I'll resolve this conversation.

Reference: https://github.com/ocornut/imgui/commit/ebe6ac5fb55aacb2a3cb0d7ee0bef7cba89cac1e#diff-cee9beb740469eb5192dfe94aaee7320932590adade9d9b1d2ee392ac2e1e88cR161